src/docs/tutorials/02-getting-started-with-flex-cli/index.md

```diff
@@ -0,0 +1,140 @@
+
+```
+flex-cli
+```
+
+This command should show you the CLI version and list available commands.
+
+## Help
+
+<script id="asciicast-267479" src="https://asciinema.org/a/267479.js" data-autoplay="true" data-speed="1.5" data-loop="1" async></script>
+
+`flex-cli help` is the command to see the list of available commands:
+
+```
+flex-cli help
+```
+
+In order to see subcommand help, pass the command as an argument for
+the `flex-cli help` command. For example, let's see help for command
+`login`.
+
+```
+flex-cli help login
+```
+
+## Get an API key
+
+To log in you need to have an API key.
+
+API keys are per marketplace and user meaning that in case you have
+two different marketplace (e.g. test marketplace and production
+marketplace) you need to have two different API keys.
+
+To get an API key, please [contact Sharetribe support
+team](mailto:flex-support@sharetribe.com) and let us know your
+marketplace ID and your admin email and. We'll generate you an API
+key. In the future, it will be possible to generate an API through
+Flex Console.
+
+## Log in
+
+<script id="asciicast-267480" src="https://asciinema.org/a/267480.js" data-autoplay="true" data-speed="1.5" data-loop="1" async></script>
+
+After you've received [your API key](#get-an-api-key), you can log in
+with `flex-cli login` command. First, let's see help for the login
+command:
+
+```
+flex-cli help login
+```
+
+You can see that the command does not require any additional
+options. So let's run it:
+
+```
+flex-cli login
+```
+
+The command will prompt you your API key.
+
+After successful log in, you will be greeted by your admin email
+address.
+
+## List processes and process versions
+
+<script id="asciicast-267481" src="https://asciinema.org/a/267481.js" data-autoplay="true" data-speed="1.5" data-loop="1" async></script>
+
+Now that you have successfully logged in and know how to use the
+`help` command, let's use CLI to list processes and process versions
+in your marketplace.
+
+The command to list processes is `process list`. Let's see the help
+first:
+
+```
+flex-cli help process list
+```
+
+As you can see, the command requires `MARKETPLACE IDENT` option. You
+can use either the long form `--marketplace=<marketplace ident here>`
+or short form `-m <marketplace ident here>`. Optionally the command
+takes `--process=PROCESS NAME` parameter to get detailed information
+about a single process.
+
+Let's list all the processes:
+
+```
+flex-cli process list -m my-test-marketplace
+```
+
+This command shows you a list of transaction processes in your marketplace.
+
+## Summary
+
+In this tutorial, we installed Flex CLI, logged in using an API key
+and tried some example commands. In addition, we familiarized
+ourselves with the `help` command that is the main source of
+documentation for the Flex CLI.
+
+Now that we know how to list processes, the next this is to [make a
+small change to the existing process](/tutorials/03-edit-transaction-process-with-flex-cli).
```

Comment:
Always add the trailing slash to internal links to avoid unnecessary redirects.

```suggestion
small change to the existing process](/tutorials/03-edit-transaction-process-with-flex-cli/).
```